### ISSUE TITLE
New fixed action button + tooltips : doesn't always work

### ISSUE THREAD

**Issue by beeb** [Dogfalo/materialize#662]
Hello,

Big up on that new release, it's really great. I'm happy to see the fixed action button has been implemented. Ideally I would like to use it in cooperation with the tooltip feature.

I noticed that the small action buttons only show up if the main (big) action button is hovered from another side that the one where the tooltip appears. In my case, I put the data-position to "left" (most logical choice for a fixed action button at the bottom right) and thus, if my mouse moves from the left onto the button, the tooltip appears but the small action buttons don't show up. If i go over to the main button from the bottom or the top, it works as expected.

Here's the code I'm using:
```
    <div class="fixed-action-btn" style="bottom: 0; right: 24px;">
        <a href="#" class="btn-floating btn-large waves-effect waves-light cyan tooltipped z-depth-2" data-position="left" data-tooltip="some text">
            <i class="large mdi-social-person-add"></i>
        </a>
        <ul>
            <li><a href="#" class="btn-floating waves-effect waves-light light-blue tooltipped z-depth-2" data-position="left" data-tooltip="some text"><i class="large mdi-social-group-add"></i></a></li>
        </ul>
    </div>
```

**Comment by acburst** [comment]
What browser are you using? this code seems to be working on my end

**Comment by Dogfalo** [comment]
We will look into this, thanks for bringing it to our attention.

**Comment by beeb** [comment]
Was in Chrome. It's a subtle bug, only happend when you cross the hidden tooltip to come onto the main fixed action button

**Comment by acburst** [comment]
Fixed in fd8d6635f4ae795773671354dc971e8ae9ca3745
